pages/recipeDetails/[recipeDetails].jsx

```diff
@@ -13,20 +17,23 @@ export async function getServerSideProps(context) {
   let recipeDocuments;
   let allergens;
 
-  const favouriteRecipes = await getFavouriteRecipes(
-    'users-list',
-    { userName: 'The User 1' },
-  );
+  const favouriteRecipes = await getFavouriteRecipes('users-list', {
```

Comment:
You've included some hard-coded data here for `userName`. 
This changes the behaviour of this component in a fundamental way.
Rather make this data an import from a file that is clearly marked as temporary.
Place this temporary file in the utils directory. 
That way it's easier to pick up where the hard-coded data is coming from, rather than sifting through the codebase to find and remove hard-coded data.

Reply:
in simple terms, this means that I should define that user in a separate file and export it from that user.

Reply:
Yes, I'm gonna try and make a dummy branch to show you what I mean. 
But I can't use my branch, lol.
You'll have to add my suggestions to your own branch.

Reply:
I did that in my branch.